javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -25,20 +25,42 @@
 
 import java.util.function.Function;
 
+import static com.github.javaparser.printer.PrettyPrinterConfiguration.IndentType.SPACES;
+import static com.github.javaparser.printer.PrettyPrinterConfiguration.IndentType.TABS;
 import static com.github.javaparser.utils.Utils.EOL;
 import static com.github.javaparser.utils.Utils.assertNotNull;
 
 /**
  * Configuration options for the {@link PrettyPrinter}.
  */
 public class PrettyPrinterConfiguration {
+    public enum IndentType {
+        /**
+         * Indent with spaces.
+         */
+        SPACES,
+
+        /**
+         * Indent with tabs as far as possible.
+         * For proper aligning, the tab width is necessary and by default 4.
+         */
+        TABS,
+
+        /**
+         * Indent with tabs but align with spaces.
```

Comment:
I'm not sure some random user will understand the difference between indenting and aligning, and how this setting will work out.

Reply:
Well, random user has a hard time to get started anyway. :-D

How do you like
```java
        /**
         * Indent with tabs but align with spaces when wrapping and aligning
         * method call chains and method call parameters.
         * 
         * <p/><i>Example result:</i>
         * <pre>
         * class Foo {
         * 
         * \tvoid bar() {
         * \t\tfoo().bar()
         * \t\t......baz(() -> {
         * \t\t..........\tboo().baa()
         * \t\t..........\t......bee(a,
         * \t\t..........\t..........b,
         * \t\t..........\t..........c);
         * \t\t..........})
         * \t\t......bam();
         * \t}
         * }
         * </pre>
         */
```
?

Reply:
Are you sure anybody wants that setting? ;-) But yeah, that's fine.

Reply:
I think so, yes.
It was like that basically before when tabs were configured.
Well, only the inital indent was in tabs and remaining stuff in spaces completely afair.
Now you have tabs for indentation and any viewer can choose the tab size he likes, while still having the call chains and parameters aligned. :-)